datasets/arxiv-2023/README.md

```diff
@@ -0,0 +1,90 @@
+# ARXIV-2023
+
+## Dataset Description
+
+arxiv-2023 is collected to be compared with ogbn-arxiv. Both datasets represent directed citation networks where each node corresponds to a paper published on arXiv and each edge indicates one paper citing another.
```

Comment:
Maybe mention this is a text attributed graph dataset where each node is associated with multiple text attributes.